sympy/stats/crv_types.py

```diff
@@ -176,6 +180,15 @@ def set(self):
     def pdf(self, x):
         return 1/(pi*sqrt((x - self.a)*(self.b - x)))
 
+    def _sample_scipy(self, size):
+        a = float(self.a)
+        scipy = import_module('scipy')
+        if scipy:
```

Comment:
No need for this. It makes the code too long. The check about scipy's existence will be put somewhere else in the future. Just assume `scipy` is imported.